internal/services/netapp/netapp_account_resource.go

```diff
@@ -102,8 +102,58 @@ func resourceNetAppAccount() *pluginsdk.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 						"organizational_unit": {
-							Type:     pluginsdk.TypeString,
-							Optional: true,
+							Type:        pluginsdk.TypeString,
+							Optional:    true,
+							Default:     "CN=Computers",
+							Description: "The Organizational Unit (OU) within the Windows Active Directory where machines will be created. If blank, defaults to 'CN=Computers'",
+						},
+						"site_name": {
+							Type:        pluginsdk.TypeString,
+							Optional:    true,
+							Default:     "Default-First-Site-Name",
```

Comment:
AFAIK it is, but I believe it is applied at the Azure API level (since the previous version of this resource still worked). I say that because this resource could *never* work if the Site value was blank, because you'd never located an AD site.

`Default-First-Site-Name` is the standard AD default for a Site, so it seemed like a good guard rail here just to be explicit.

Reply:
Double-checking this, it is indeed the default

https://learn.microsoft.com/en-us/azure/azure-netapp-files/create-active-directory-connections#create-an-active-directory-connection

> - AD Site Name (required)
This is the AD DS site name that will be used by Azure NetApp Files for domain controller discovery.

> The default site name for both AD DS and Microsoft Entra Domain Services is Default-First-Site-Name. Follow the [naming conventions for site names](https://learn.microsoft.com/en-us/troubleshoot/windows-server/identity/naming-conventions-for-computer-domain-site-ou#site-names) if you want to rename the site name.